assets/src/components/map/tooltip.tsx

```diff
@@ -6,17 +6,15 @@ interface MapTooltipProps {
   permanent?: boolean
 }
 
-export const MapTooltip = ({ title, permanent }: MapTooltipProps) => {
-  return (
-    <Tooltip
-      key={`tooltip-permanent-${permanent}`}
-      className="c-detour_map--tooltip"
-      direction="top"
-      offset={[0, -15]}
-      sticky
-      permanent={permanent}
-    >
-      {title}
-    </Tooltip>
-  )
-}
+export const MapTooltip = ({ title, permanent }: MapTooltipProps) => (
+  <Tooltip
+    key={`tooltip-permanent-${permanent}`}
+    className="c-map_tooltip"
```

Comment:
nit: I would name this using dashes instead of underlines
```suggestion
    className="c-map-tooltip"
```

Reply:
https://github.com/mbta/skate/pull/2366/commits/0510b487eced731c65607671ae33452a40f15120

Reply:
Thanks!